tests/utils.py

```diff
@@ -1,24 +1,13 @@
 from __future__ import annotations
 
 import datetime
-from functools import wraps
 from pathlib import Path
 from typing import Any, get_args
 
-import pytest
 import yaml
 from pydantic import BaseModel
 
 
-def inside_rms(func):
```

Comment:
Since we only have one fixture to apply now, I removed this layer of abstraction and use that fixture directly in the tests instead.